### PR TITLE
LPS-68589 Apply to MB and DL (should be applied to all cases)

### PR DESCRIPTION
**Pull request by robertoDiaz**

### CODE REVIEW

**Comment by sergiogonzalez** [comment]
ci:reopen

File: portal-impl/src/com/liferay/portlet/messageboards/service/impl/MBMessageLocalServiceImpl.java [hunk 1]
```diff
@@ -2042,13 +2044,14 @@ protected String getSubject(String subject, String body) {
...
 
 	protected MBSubscriptionSender getSubscriptionSender(
-		long userId, MBMessage message, String messageURL, String entryTitle,
-		boolean htmlFormat, String messageBody, String categoryName,
-		String inReplyTo, String fromName, String fromAddress,
-		String replyToAddress, String emailAddress, String fullName,
-		LocalizedValuesMap subjectLocalizedValuesMap,
-		LocalizedValuesMap bodyLocalizedValuesMap,
-		ServiceContext serviceContext) {
+			long userId, Group group, MBMessage message, String messageURL,
```
Comment (robertoDiaz) [inline portal-impl/src/com/liferay/portlet/messageboards/service/impl/MBMessageLocalServiceImpl.java]: No cambiar notacion del metodo

File: portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLAppHelperLocalServiceImpl.java [hunk 2]
```diff
@@ -1718,12 +1718,16 @@ else if (commandUpdate &&
 		subscriptionSender.setClassPK(fileVersion.getFileEntryId());
 		subscriptionSender.setClassName(DLFileEntryConstants.getClassName());
 		subscriptionSender.setCompanyId(fileVersion.getCompanyId());
+
+		if (folderId != DLFolderConstants.DEFAULT_PARENT_FOLDER_ID) {
```
Comment (robertoDiaz) [inline portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLAppHelperLocalServiceImpl.java]: Unir condicionales con if else

File: portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLAppHelperLocalServiceImpl.java [hunk 3]
```diff
@@ -1733,6 +1737,17 @@ else if (commandUpdate &&
...
 		subscriptionSender.setLocalizedBodyMap(
 			LocalizationUtil.getMap(bodyLocalizedValuesMap));
+
+		if (folderId != DLFolderConstants.DEFAULT_PARENT_FOLDER_ID) {
+			subscriptionSender.setLocalizedContextFunctionAttribute(
+				"[$FOLDER_NAME$]",
+				(Locale locale) -> LanguageUtil.get(locale, "home"));
+		}
+
+		subscriptionSender.setLocalizedContextFunctionAttribute(
```
Comment (robertoDiaz) [inline portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLAppHelperLocalServiceImpl.java]: Escapar por defecto + añadir atributo para evitarlo si es necesario

**Comment by sergiogonzalez** [comment]
Closing the pull for now since we have already discussed the concerns and we have commented about them.

**Pull request closed without merging**